locales/en-US/strings.properties

```diff
@@ -143,6 +143,10 @@ pocket_read_more=Popular Topics:
 # LOCALIZATION NOTE (pocket_read_even_more): This is shown as a link at the
 # end of the list of popular topic links.
 pocket_read_even_more=View More Stories
+pocket_more_reccommendations=More Recommendations
+pocket_learn_more=Learn More
+pocket_cta_button=Get Pocket
+pocket_cta_text=Save the stories you love in Pocket, and fuel your mind with fascinating reads.
```

Comment:
@Mardak I'm told I need to reach out to you about making these strings marked for marketing. I'm told some sort of comment or annotation needs to be added?

Reply:
I'm not really sure what specially needs to be done for marketing? Unless you're asking if these strings are finalized for use in product. @aaronrbenson has worked with people to get approval on specific wording of strings.

Reply:
@Mardak The marketing team would like to provide the translations for this string, but we aren't sure what the right process is. For example we have the string we would like to use in German but don't exactly know how to manage putting it in the right place.

Reply:
Ah... @flodolo ? How do we get specific german marketing strings into pontoon?

Reply:
@flodolo thanks, super helpful. The uplift you mentioned is just uplifting the changes to the .properties file, right?

Also, aside from filing a bug/setting up a call, is there something that needs to be added to the code here and now? I was under the impression there is something I would still need to do even on top of a bug and call?

Reply:
Nothing to add here, because it would show for 100 languages, and confuse them (I assume this string will be only used for German, but it's still exposed to all language we ship Firefox in).

As for uplift: it would require a bit more, given how Activity Stream is built. They'd need to build an update version, possibly only for beta or release, and uplift that code.